chanzuckerberg/idseq-web
# Rake task to create new alignment config

# Description

This is a necessary step for using newly generated indexes in IDseq. The rake task simplifies the step and checks that the files exist. 

# Tests

* run on existing files `NCBI_DATE=2019-09-17 rails create_alignment_config`, see save message
* run on non-existing files `NCBI_DATE=2019-09-18 rails create_alignment_config`, see error message
